module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,7 +1,7 @@
 #include <array>
```

Comment:
\<array\> already in 'compression.hpp' thus doubled here

Reply:
It's not a part of the home work....

Reply:
You always need to check includes. When you will work with other colleague in future work, and you make changes, you are responsible for whole code, not only your implementation :)